backend/Dockerfile.scheduledworkflow

```diff
@@ -45,6 +45,7 @@ COPY --from=builder /tmp/licenses.csv /third_party/licenses.csv
 COPY --from=builder /tmp/NOTICES /third_party/NOTICES
 
 ENV NAMESPACE ""
+ENV EXECUTIONTYPE Workflow
 ENV LOG_LEVEL info
 
-CMD /bin/controller --logtostderr=true --namespace=${NAMESPACE} --logLevel=${LOG_LEVEL}
+CMD /bin/controller --logtostderr=true --namespace=${NAMESPACE} --executionType ${EXECUTIONTYPE} --logLevel=${LOG_LEVEL}
```

Comment:
@rimolive you can revert this change because we removed the new `executionType` flag in scheduled workflow in this PR.